beacon-chain/slasher/receive.go

```diff
@@ -218,9 +215,11 @@ func (s *Service) pruneSlasherDataWithinSlidingWindow(ctx context.Context, curre
 	if err != nil {
 		return errors.Wrap(err, "Could not prune proposals")
 	}
-	log.WithFields(logrus.Fields{
-		"prunedAttestationRecords": numPrunedAtts,
-		"prunedProposalRecords":    numPrunedProposals,
-	}).Info("Successfully pruned slasher data")
+	if numPrunedAtts > 0 || numPrunedProposals > 0 {
```

Comment:
imo: always log this, whether something was pruned or not. It helps to see that pruning is working and completing task successfully.

Optional: Add the prune time duration as a field, that might look nice for debugging or performance monitoring. Other logs reference "elapsed" as field name.